core/tee/tee_svc_cryp.c

```diff
@@ -427,17 +480,33 @@ static const struct tee_cryp_obj_type_props tee_cryp_obj_props[] = {
 		sizeof(struct rsa_keypair),
 		tee_cryp_obj_rsa_keypair_attrs),
 
-	PROP(TEE_TYPE_DSA_PUBLIC_KEY, 64, 512, 1024,
+	PROP(TEE_TYPE_DSA_PUBLIC_KEY, 64, 512, 3072,
```

Comment:
With such a definition, will we accept a key size of 1024+64=1088, which should not be the case?